packages/cli/src/databases/migrations/common/1674509946020-CreateLdapEntities.ts

```diff
@@ -0,0 +1,71 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+import { LDAP_DEFAULT_CONFIGURATION, LDAP_FEATURE_NAME } from '@/Ldap/constants';
+
+export class CreateLdapEntities1674509946020 implements ReversibleMigration {
+	async up({ escape, dbType, isMysql, executeQuery }: MigrationContext) {
+		const userTable = escape.tableName('user');
+		await executeQuery(
+			`ALTER TABLE ${userTable} ADD COLUMN disabled BOOLEAN NOT NULL DEFAULT false;`,
+		);
+
+		await executeQuery(`
+			INSERT INTO ${escape.tableName('settings')} (${escape.columnName(
+			'key',
+		)}, value, ${escape.columnName('loadOnStartup')})
+			VALUES ('${LDAP_FEATURE_NAME}', '${JSON.stringify(LDAP_DEFAULT_CONFIGURATION)}', true)
+		`);
+
+		const uuidColumnType = dbType === 'postgresdb' ? 'UUID' : 'VARCHAR(36)';
+
+		await executeQuery(
+			`CREATE TABLE IF NOT EXISTS ${escape.tableName('auth_identity')} (
+				${escape.columnName('userId')} ${uuidColumnType} REFERENCES ${userTable} (id),
+				${escape.columnName('providerId')} VARCHAR(64) NOT NULL,
+				${escape.columnName('providerType')} VARCHAR(32) NOT NULL,
+				${escape.columnName('createdAt')} timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+				${escape.columnName('updatedAt')} timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+				PRIMARY KEY(${escape.columnName('providerId')}, ${escape.columnName('providerType')})
```

Comment:
Subjective, but I find the helpers are hurting readability. Maybe escape first and then use the escaped values in the template literal?